R/generate_wrappers.R

```diff
@@ -8,22 +8,18 @@ generateNoNoiseRawData <- function(cell_lines, drugs, save = TRUE) {
   #   only for testing purpuses not displayed as example
   df_merged <- prepareMergedData(cell_lines[2:11, ], drugs[2:11, ], 0)
   
-  if (requireNamespace("gDRcore", quietly = TRUE)) {
-    mae <- gDRcore::runDrugResponseProcessingPipeline(
-      df_merged,
-      nested_confounders = gDRutils::get_env_identifiers("barcode")[1]
+  mae <- runDrugResponseProcessingPipeline(
+    df_merged,
+    nested_confounders = gDRutils::get_env_identifiers("barcode")[1]
+  )
+  
+  if (save) {
+    save_rds(
+      rdsObj = mae,
+      rdsName = "finalMAE_small_no_noise.RDS"
     )
-    
-    if (save) {
-      save_rds(
-        rdsObj = mae,
-        rdsName = "finalMAE_small_no_noise.RDS"
-      )
-    }
-    invisible(mae)
-  } else {
-    df_merged
   }
```

Comment:
```
} else {
    df_merged
  }
```
Was it intentionally deleted?

Reply:
Yes, because I removed ` if (requireNamespace("gDRcore", quietly = TRUE))` that is always true in this package

Reply:
Thx  😄